### DISCUSSION TITLE
How to export IBNR_ to SQL

### DISCUSSION THREAD

**Discussion by kydoshine** [casact/chainladder-python#253] [Q&A]
Hi All,

I am trying to export the monthly IBNR estimates to SQL, but I can't figure out if the .IBNR_ outputs as a multi-column dataframe or if it is a single row that I need to split? I tried to get the column names, but I just get an error that says '0' is not in the list. Any help would be greatly appreciated!

**Comment by kennethshsu** [comment]
I'm not sure how your database is structured, but I assume that you can push a csv to your SQL databases. 

Did you call `.to_frame()` correctly? 

```python
import pandas as pd
import chainladder as cl

raa = cl.load_sample("raa")
df = cl.Chainladder().fit(raa).ibnr_.to_frame()
df.to_csv(...)
```

In the above example, we have a single column data frame. 

Can you post your code using the sample data?
